vtm.js

```diff
@@ -359,6 +359,44 @@ const attributesAndSkillsMixin = {
   },
 }
 
+/**
+ * displays state of current resource assigment
+ * there are two input arrays
+ * allocatedResources array contains number of attributes assigned to that value
+ * resourceRestrictions array contains number of attributes allowed
+ * this component will display number of allocated points vs. allowed
+ * */
+app.component('restriction-state', {
```

Comment:
dost som rozmyslal nad tym ci to fakt ma byt komponenta a nie nieco ine, ale jednak neviem co a podruhe nemam nejake padne argumenty preco nie komponenta